backend/controllers/user.js

```diff
@@ -48,3 +48,31 @@ module.exports.getAllUsers = async (req, res) => {
     });
   }
 };
+
+module.exports.updatePassword = async (req, res) => {
```

Comment:
Do we need to check if the new password = old password?

Reply:
We could give it as a task to the students 😅